tests/net/ip-addr/src/main.c

```diff
@@ -602,6 +602,104 @@ ZTEST(ip_addr_fn, test_ipv6_mesh_addresses)
 		     "IPv6 removing address failed\n");
 }
 
+ZTEST(ip_addr_fn, test_private_ipv6_addresses)
+{
+	bool ret;
+	struct {
+		struct in6_addr addr;
+		bool is_private;
+	} addrs[] = {
+		{
+			.addr = { { { 0x20, 0x01, 0x0d, 0xb8, 0, 0, 0, 0,
+						0, 0, 0, 0, 0, 0, 0x99, 0x1 } } },
```

Comment:
This indentation looks a bit weird, I don't think it was intended? Same in others below.